src/test/java/bio/terra/service/duos/DuosDaoTest.java

```diff
@@ -53,26 +54,26 @@ public class DuosDaoTest {
   public void before() {
     duosDao = new DuosDao(jdbcTemplate, TDR_SERVICE_ACCOUNT_EMAIL);
 
-    duosId = UUID.randomUUID().toString();
-    firecloudGroupName = String.format("%s_users", duosId);
+    duosFirecloudGroupIds = new ArrayList<>();
+    firecloudGroupName = String.format("%s_users", DUOS_ID);
     firecloudGroupEmail = String.format("%s@dev.test.firecloud.org", firecloudGroupName);
     toInsert =
         new DuosFirecloudGroupModel()
-            .duosId(duosId)
+            .duosId(DUOS_ID)
             .firecloudGroupName(firecloudGroupName)
             .firecloudGroupEmail(firecloudGroupEmail);
   }
 
   @After
   public void after() {
-    if (duosFirecloudGroupId != null) {
+    for (UUID duosFirecloudGroupId : duosFirecloudGroupIds) {
       assertTrue(duosDao.deleteFirecloudGroup(duosFirecloudGroupId));
```

Comment:
just in the interest of debugging, it might be nice to have a message here like 
```suggestion
      assertTrue("duos group " + duosFirecloudGroupId + " was deleted", duosDao.deleteFirecloudGroup(duosFirecloudGroupId));
```

Reply:
any maybe log somewhere in the test the id vs the group name (so we can figure which one was the problem in the test)

Reply:
Good ideas!

Reply:
Here's the updated logging output when running `testUpdateFirecloudGroupsLastSyncedDate`, it's primarily coming from `DuosDao`:

```
2023-02-10 18:09:49,231 INFO  [Test worker] bio.terra.service.duos.DuosDao: Inserted DUOS-ID0 -> DUOS-ID_users into duos_firecloud_group with id 76def1c3-fa6e-45d7-8649-a816595da2a0
2023-02-10 18:09:49,240 INFO  [Test worker] bio.terra.service.duos.DuosDao: Inserted DUOS-ID1 -> DUOS-ID_users into duos_firecloud_group with id 9d89cbbc-b701-447b-9cf1-75512c0c694f
2023-02-10 18:09:49,244 INFO  [Test worker] bio.terra.service.duos.DuosDao: Inserted DUOS-ID2 -> DUOS-ID_users into duos_firecloud_group with id f387585b-3e77-4a77-8d6b-296b306bbecd
2023-02-10 18:09:49,254 INFO  [Test worker] bio.terra.service.duos.DuosDao: Updating 3 Firecloud group record(s) last synced date to 2022-11-17T00:00:00Z
2023-02-10 18:09:49,268 INFO  [Test worker] bio.terra.service.duos.DuosDao: Deleting Firecloud group record 76def1c3-fa6e-45d7-8649-a816595da2a0
2023-02-10 18:09:49,271 INFO  [Test worker] bio.terra.service.duos.DuosDao: Deleting Firecloud group record 9d89cbbc-b701-447b-9cf1-75512c0c694f
2023-02-10 18:09:49,273 INFO  [Test worker] bio.terra.service.duos.DuosDao: Deleting Firecloud group record f387585b-3e77-4a77-8d6b-296b306bbecd
```

Reply:
love it!